### ISSUE TITLE
support nested blocks

### ISSUE THREAD

**Issue by notslang** [pabluk/tumblr-theme-parser#2]
it looks like @martinpucala has done some work on it over here: https://github.com/martinpucala/tumblr-theme-parser ... maybe that can be merged in and used?

**Comment by pabluk** [comment]
@slang800 yes, it seems a very good idea, i'll review the commits from the fork of @martinpucala to merge them.

**Comment by pabluk** [comment]
It will not be easy to merge, because this commit https://github.com/martinpucala/tumblr-theme-parser/commit/3308e73654244b08dee4a9f7ff48c3e1028f8f29 broke many tests and @martinpucala added a warning about this in https://github.com/martinpucala/tumblr-theme-parser/commit/4f92e97a6226d73ab6e7624963c4d59e6ec1fb60

**Comment by notslang** [comment]
Update on this: I ended up writing [my own parser](https://www.npmjs.com/package/tumblr-theme-parser) in JS. Idk if that's at all useful to you, but I wanted to let you know just in case.

**Comment by pabluk** [comment]
@slang800 great, your parser looks very good!
